habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -1247,8 +1247,8 @@ class AgentConfig(HabitatBaseConfig):
     start_position: List[float] = [0, 0, 0]
     start_rotation: List[float] = [0, 0, 0, 1]
     joint_start_noise: float = 0.1
-    robot_urdf: str = "data/robots/hab_fetch/robots/hab_fetch.urdf"
-    robot_type: str = "FetchRobot"
+    articulated_agent_urdf: str = "data/robots/hab_fetch/robots/hab_fetch.urdf"
```

Comment:
I would keep `articulated_agent` for type and urdf.

Reply:
The current change reflects it, right?

Reply:
yep, I meant in regards to the comment that we should remove articulated_agent from observations